shared/oae/js/jquery-plugins/jquery.switchtab.js

```diff
@@ -0,0 +1,166 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/**
+ * Utility plugin that handles the "switch tab" component interactions.
+ */
+
+define(['jquery', 'jquery.history'], function (jQuery, oaeUtil) {
+    (function($) {
+
+        /**
+         * OAE plugin that implements the interaction of the "Switch Tab" component. A Switch Tab
+         * is essentially a set of tabs that look like a left-to-right switch. There can only be
+         * one selected at one time. Back functionality is also provided.
+         *
+         * Usage:
+         *
+         * You should create a template that is based on the bootstrap pill component, with the
+         * one exception that the root element has the `oae-switchtab` class:
+         *
+         *  ```
```

Comment:
I would still prefer this documentation to be added to `oae.components.css` instead. `oae.components.css` is where you can find the documentation for all other component, and the goal is to automatically parse `oae.components.css` and turn it into a widget development component guide at some point